sig-auth/README.md

```diff
@@ -134,7 +128,6 @@ Node identity management (co-owned with sig-lifecycle), and authorization restri
 ### policy-management
 API validation and policies enforced during admission, such as PodSecurityPolicy. Excludes run-time policies like NetworkPolicy and Seccomp.
 - **Owners:**
-  - [kubernetes-sigs/wg-policy-prototypes](https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/OWNERS)
```

Comment:
@JimBugwadia could you open an issue to archive https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master?

example issue: https://github.com/kubernetes/org/issues/5484

Reply:
there is an issue template for archival: https://github.com/kubernetes/org/blob/main/.github/ISSUE_TEMPLATE/repo-archive.yml

Reply:
Done! https://github.com/kubernetes/org/issues/5658